composer.json

```diff
@@ -22,11 +22,12 @@
   "minimum-stability": "dev",
   "prefer-stable": true,
   "require": {
-    "drunomics/contentpool_data_model": "^1.0.2",
+    "drunomics/contentpool_data_model": "^1.0.3",
     "drunomics/contentpool_replication": "^1.0@dev",
     "drunomics/service-utils": "*",
     "drupal/core": "^8.5.6",
     "drupal/entity_edit_redirect": "^1.5",
+    "drupal/markup_field": "^1.0@alpha",
```

Comment:
This should be added in by teh data model dependency already, let'S avoid duplicating dependencies here.